src/main/java/warmup/Arizona.java

```diff
@@ -0,0 +1,135 @@
+package main.java.warmup;
+// Here are warmup-2 practice task storred;
+
+public class Arizona {
+    public static void main(String[] args) {
+        System.out.println("Arizona");
+    }
+
+    //stringTimes
+    public String stringTimes(String str, int n) {
+        if (str == null) {
+            return str;
+        }
+        String stringTimes = "";
+        for (int i = 0; i < n; i++) {
+            stringTimes += str;
+        }
+        return stringTimes;
+    }
+
+    //frontTimes
+    public String frontTimes(String str, int n) {
+        if (str == null) {
+            return str;
+        }
+        String result = "";
+        for (int i = 0; i < n; i++) {
+            if (str.length() >= 3) {
+                result += str.substring(0, 3);
+            } else {
+                result += str;
+            }
+        }
+        return str;
+    }
+
+    //countXX
+    int countXX(String str) {
+        if (str == null) {
+            return 0;
+        }
+        int count = 0;
+        for (int i = 0; i < str.length() - 1; i++) {
+            if (str.charAt(i) == 'x') {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    //double X
+    boolean doubleX(String str) {
+        if (str == null) {
+            return false;
+        }
+        int x = str.indexOf("x");
```

Comment:
use:
'x'
instead of 
"x"
indexOf(char) is faster than indexOf(String) for a single character.